helm/prometheus/locals.tf

```diff
@@ -0,0 +1,19 @@
+locals {
+  helm_stable_repository = "https://kubernetes-charts.storage.googleapis.com"
+
+  storage_class_defaults = {
+    "aws" = "gp2"
+    "gcp" = "standard"
+  }
+  storage_class = merge(local.storage_class_defaults, var.storage_class)
+
+  prometheus_helm_defaults = {
+    "server.persistentVolume.storageClass"       = lookup(local.storage_class, var.cloud_provider, "gp2")
+    "server.persistentVolume.size"               = "500Gi" # Without any idea of what normal is I'm just setting this as a random value
```

Comment:
this number has been vetted so we should remove the "random value" comment :)

Reply:
Removed in fb944627124ced1dd161a4955368e2d8d81ef54a